python-inocybe-openswitch/models/base-env-tempctl.yang

```diff
@@ -0,0 +1,152 @@
+module base-env-tempctl {
+  namespace "http://www.dellemc.com/networking/os10/dell-base-env-tempctl";
+  prefix base-env-tempctl;
+
+  import base-platform-common {
+    prefix platform;
+  }
+
+  organization "Dell EMC";
+  contact
+    "http://www.dell.com/support";
+  description
+    "Environmental temperature control service manages various aspects including:
+     1) fan speed control (FSC),
+     2) temperature fault events handling,
+     3) logging of events and faults,
+     4) exception handling.
+
+     This model implements the configuration and Operational Data
+     related to this service.
+
+     Copyright (c) 2015-2018 by Dell EMC, All rights reserved.";
+
+  revision 2018-03-30 {
```

Comment:
Since this is a new file, wondering if you need to get rid of all the previous references, and just put on with the latest date and the description as "Initial Version".  Upto you.

But in any case, you would need to put the latest revision.